astronomer/providers/amazon/aws/operators/redshift_data.py

```diff
@@ -29,47 +17,14 @@ def __init__(
         poll_interval: int = 5,
         **kwargs: Any,
     ) -> None:
-        self.poll_interval = poll_interval
-        super().__init__(**kwargs)
-
-    def execute(self, context: Context) -> None:
-        """
-        Makes a sync call to RedshiftDataHook, executes the query and gets back the list of query_ids and
-        defers trigger to poll for the status for the queries executed.
-        """
-        redshift_data_hook = RedshiftDataHook(aws_conn_id=self.aws_conn_id)
-        query_ids, response = redshift_data_hook.execute_query(sql=self.sql, params=self.params)
-        self.log.info("Query IDs %s", query_ids)
-        if response.get("status") == "error":
-            self.execute_complete(context, event=response)
-        context["ti"].xcom_push(key="return_value", value=query_ids)
-
-        if redshift_data_hook.queries_are_completed(query_ids, context):
-            self.log.info("%s completed successfully.", self.task_id)
-            return
-
-        self.defer(
-            timeout=self.execution_timeout,
-            trigger=RedshiftDataTrigger(
-                task_id=self.task_id,
-                poll_interval=self.poll_interval,
-                aws_conn_id=self.aws_conn_id,
-                query_ids=query_ids,
+        warnings.warn(
+            (
+                "This module is deprecated and will be removed in 2.0.0."
+                "Please use `airflow.providers.amazon.aws.operators.redshift_data.RedshiftDataOperator`"
+                "and set `deferrable` param to `True` instead."
             ),
-            method_name="execute_complete",
+            DeprecationWarning,
+            stacklevel=2,
         )
-
-    def execute_complete(self, context: Context, event: Any = None) -> None:
-        """
-        Callback for when the trigger fires - returns immediately.
-        Relies on trigger to throw an exception, otherwise it assumes execution was
-        successful.
-        """
-        if event:
-            if "status" in event and event["status"] == "error":
-                msg = "context: {}, error message: {}".format(context, event["message"])
-                raise AirflowException(msg)
-            elif "status" in event and event["status"] == "success":
-                self.log.info("%s completed successfully.", self.task_id)
-        else:
-            raise AirflowException("Did not receive valid event from the trigerrer")
+        kwargs["poll_interval"] = poll_interval
```

Comment:
do we need this line? 
looks like `poll_interval ` is kwargs we can just remove this from the init